spec/dummy/app/views/pages/turbo_frame_tag_hello_world.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= turbo_frame_tag 'hello-turbo-stream' do %>
+    <%= button_to "send me hello-turbo-stream component", turbo_stream_send_hello_world_path %>
+<% end %>
```

Comment:
please add return

Reply:
hi, i don't get it! what should i 'return' here. some document like other pages, right ?